mongo/cursor.go

```diff
@@ -167,24 +152,27 @@ func (c *Cursor) next(ctx context.Context, nonBlocking bool) bool {
 	}
 }
 
-// Decode will decode the current document into val. If val is nil or is a typed nil, an error will be returned.
+// Decode will unmarshal the current document into val and return any errors from the unmarshalling process without any
+// modification. If val is nil or is a typed nil, an error will be returned.
 func (c *Cursor) Decode(val interface{}) error {
 	return bson.UnmarshalWithRegistry(c.registry, c.Current, val)
 }
 
-// Err returns the current error.
+// Err returns the last error seen by the Cursor, or nil if no errors has occurred.
 func (c *Cursor) Err() error { return c.err }
 
-// Close closes this cursor.
+// Close closes this cursor. Next and TryNext must not be called to get more documents from the cursor after Close
```

Comment:
Same as changestream. This can be "Next and TryNext must not be called after Close has been called"

Reply:
done.